### DISCUSSION TITLE
Is the configuration of external secrets via SECRET[backend_aws_secrets_manager.*] universally supported?

### DISCUSSION THREAD

**Discussion by tbenade** [vectordotdev/vector#20088] [Q&A]
Hi,

Trying to use the external secrets placeholder/backend with little luck in the datadog_metrics sink. The secrets retrieval works great for our Splunk sink but for 'datadog_metrics' we only get the `403 fiobidden`. If I replace `SECRET[backend_aws_secrets_manager.datadog_api_key]` with the value <datadog api key> stored in AWS secrets manager it works fine.

```
type: datadog_metrics
inputs:
  - internal_metrics
default_api_key: "SECRET[backend_aws_secrets_manager.datadog_api_key]"
default_namespace: vector-aggregator-service
```

Before I burn much more time on this thought I would check that parsing the `SECRET[*.*]` was actually supported in the datadog_metrics sink.

Thanks

**Comment by jszwedko** [comment]
It's supported everywhere as it is actually handled before configuration parsing by simple string templating.
